### ISSUE TITLE
Incompatibility with Docker 19.03.12

### ISSUE THREAD

**Issue by yuchaoran2011** [marcuslonnberg/sbt-docker#111]
Unable to build Docker images:
```
[info] unknown shorthand flag: 'f' in -f
[error] java.lang.RuntimeException: Nonzero exit value: 1
[error] 	at scala.sys.package$.error(package.scala:30)
[error] 	at scala.sys.process.ProcessBuilderImpl$AbstractBuilder.slurp(ProcessBuilderImpl.scala:138)
[error] 	at scala.sys.process.ProcessBuilderImpl$AbstractBuilder.$bang$bang(ProcessBuilderImpl.scala:109)
[error] 	at sbtdocker.DockerVersion$.apply(DockerVersion.scala:19)
[error] 	at sbtdocker.DockerTag$.apply(DockerTag.scala:17)
[error] 	at sbtdocker.DockerBuild$.$anonfun$buildAndTag$5(DockerBuild.scala:75)
[error] 	at sbtdocker.DockerBuild$.$anonfun$buildAndTag$5$adapted(DockerBuild.scala:74)
[error] 	at scala.collection.immutable.List.foreach(List.scala:392)
[error] 	at sbtdocker.DockerBuild$.buildAndTag(DockerBuild.scala:74)
[error] 	at sbtdocker.DockerBuild$.apply(DockerBuild.scala:45)
[error] 	at sbtdocker.DockerBuild$.apply(DockerBuild.scala:24)
[error] 	at sbtdocker.DockerSettings$.$anonfun$baseDockerSettings$1(DockerSettings.scala:17)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:67)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:281)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:19)
[error] 	at sbt.Execute.work(Execute.scala:290)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:281)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[error] 	at java.lang.Thread.run(Thread.java:748)
[error] (docker) Nonzero exit value: 1
[error] Total time: 24 s, completed Jun 28, 2020 8:50:52 PM
```
My Docker version is:
```
$ docker version
Client: Docker Engine - Community
 Azure integration  0.1.4
 Version:           19.03.12
 API version:       1.40
 Go version:        go1.13.10
 Git commit:        48a66213fe
 Built:             Mon Jun 22 15:41:33 2020
 OS/Arch:           darwin/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.12
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.10
  Git commit:       48a66213fe
  Built:            Mon Jun 22 15:49:27 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.2.13
  GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```
The above error only started to happen after the recent Docker upgrade.

**Comment by yuchaoran2011** [comment]
Verified that by downgrading Docker to 19.03.8, it works again:
```
$ docker version
Client: Docker Engine - Community
 Version:           19.03.8
 API version:       1.40
 Go version:        go1.12.17
 Git commit:        afacb8b
 Built:             Wed Mar 11 01:21:11 2020
 OS/Arch:           darwin/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.8
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.17
  Git commit:       afacb8b
  Built:            Wed Mar 11 01:29:16 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.2.13
  GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

**Comment by marcuslonnberg** [comment]
Thanks for reporting this! I'm aiming to have a fix available tomorrow.

**Comment by yuchaoran2011** [comment]
@marcuslonnberg Awesome! Thanks for getting to it quickly.

**Comment by marcuslonnberg** [comment]
A fix for this have now been released in version 1.6.0.
